pkg/unvanquished_src.dpkdir/ui/options_graphics.rml

```diff
@@ -105,10 +105,15 @@
 					<p><translate>Precomputed dynamic lights emitted by static map objects.</translate></p>
 				</row>
 				<row>
-					<h3><translate>Dynamic lights</translate></h3>
+					<h3><translate>Realtime dynamic lights</translate></h3>
 					<input cvar="r_dynamicLight" type="checkbox" />
 					<p><translate>Realtime dynamic lights emitted by moving objects like weapons and buildings.</translate></p>
 				</row>
+				<row>
+					<h3><translate>Realtime static lights</translate></h3>
```

Comment:
In fact the need was to avoid building the non-static dynamic light renderer.

I have a better solution for that:

https://github.com/DaemonEngine/Daemon/pull/1234

Reply:
The new cvar r_realtimeLighting` is now used and `r_dynamicLight` and `r_staticLight` are not modified anymore.